src/Analysis/GWASApp/Components/AttritionTableWrapper/AttritionTable/AttritionTableModal/AttritionTableModal.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Modal } from 'antd';
+import './AttritionTableModal.css';
+import PhenotypeHistogram from '../../../Diagrams/PhenotypeHistogram/PhenotypeHistogram';
+import CohortsOverlapDiagram from '../../../Diagrams/CohortsOverlapDiagram/CohortsOverlapDiagram';
+
+const AttritionTableModal = ({ modalInfo, setModalInfo }) => {
+  const modalWidth = 650;
+
+  return (
+    <Modal
+      title={<h3>{modalInfo.title}</h3>}
+      open={modalInfo.isModalOpen}
+      onOk={() => setModalInfo({ ...modalInfo, isModalOpen: false })}
+      onCancel={() => setModalInfo({ ...modalInfo, isModalOpen: false })}
+      footer={null}
+      width={modalWidth}
+      className='attrition-table-modal'
+    >
+      {modalInfo?.rowObject
+        && modalInfo.rowObject.variable_type === 'concept' && (
+        <div data-testid='phenotype-histogram-diagram'>
+          <PhenotypeHistogram
+            useInlineErrorMessages
+            selectedStudyPopulationCohort={modalInfo.selectedCohort}
+            selectedCovariates={
+              modalInfo.currentCovariateAndCovariatesFromPrecedingRows
+            }
+            outcome={modalInfo.outcome}
+            selectedContinuousItem={modalInfo.rowObject}
+            useAnimation={false}
+          />
+        </div>
+      )}
+      {modalInfo?.rowObject
+        && modalInfo.rowObject.variable_type === 'custom_dichotomous' && (
+        <div data-testid='euler-diagram'>
+          <CohortsOverlapDiagram
+            useInlineErrorMessages
+            selectedStudyPopulationCohort={modalInfo.selectedCohort}
+            selectedCaseCohort={{
+              cohort_name: modalInfo?.rowObject?.cohort_names[0],
+              size: modalInfo?.rowObject?.cohort_sizes[0],
+              cohort_definition_id: modalInfo?.rowObject?.cohort_ids[0],
+            }}
+            selectedControlCohort={{
+              cohort_name: modalInfo?.rowObject?.cohort_names[1],
+              size: modalInfo?.rowObject?.cohort_sizes[1],
+              cohort_definition_id: modalInfo?.rowObject?.cohort_ids[1],
+            }}
+            // Need to remove first item from currentCovariateAndCovariatesFromPrecedingRows for this visualization
+            selectedCovariates={modalInfo.currentCovariateAndCovariatesFromPrecedingRows.slice(
+              1,
+            )}
```

Comment:
as discussed, this removal should only happen for the visualization of the "outcome" row diagram

Reply:
This has been updated to use a variable to determine if the row is the outcome:

` const rowIsOutcome =  modalInfo.currentCovariateAndCovariatesFromPrecedingRows?.length === 1;`

Then the selectedCovariates and outcome props for the CohortsOverlapDiagram use this variable to determine the values: 

```
            selectedCovariates={rowIsOutcome ? [] : modalInfo.currentCovariateAndCovariatesFromPrecedingRows}
            outcome={rowIsOutcome ? null : modalInfo.outcome}
```